Mathlib/Order/InitialSeg.lean

```diff
@@ -27,6 +27,11 @@ These notations belong to the `InitialSeg` locale.
 
 * `r ≼i s`: the type of initial segment embeddings of `r` into `s`.
 * `r ≺i s`: the type of principal segment embeddings of `r` into `s`.
+* `α ≤i β` is an abbreviation for `(· < ·) ≼i (· < ·)`.
+* `α <i β` is an abbreviation for `(· < ·) ≺i (· < ·)`.
+
+Due to `InitialSeg.ltOrEq`, we can think of principal segment embeddings as a "strict" version of
+initial segment embeddings.
```

Comment:
This doesn't convince me hugely. `r a b -> s a b or a = b` doesn't really mean that `r` is `<=` and `s` is `<`. For example, `s` could well be reflexive.

More simply, don't initial and principal segment embeddings correspond to `<=` and `<` on ordinals?

Reply:
Yeah, that works too.